packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -5,66 +5,98 @@
 
 'use strict';
 
-const { commitTypes, transform } = require( '../changelog/writer-options' );
+const conventionalCommitsParser = require( 'conventional-commits-parser' );
+const conventionalCommitsFilter = require( 'conventional-commits-filter' );
+const gitRawCommits = require( 'git-raw-commits' );
+const concat = require( 'concat-stream' );
+const parserOptions = require( './parser-options' );
+const { availableCommitTypes } = require( './transform-commit-utils' );
+const getPackageJson = require( './getpackagejson' );
+const versions = require( './versions' );
 
 /**
  * Returns a type (major, minor, patch) of the next release based on commits.
  *
+ * If given package has not changed, suggested version will be equal to 'skip'.
+ *
+ * @param {Function} transformCommit
+ * @param {Boolean} isDevPackage Is the function called in a repository
+ * with multiple packages (which is management by Lerna).
  * @returns {Promise}
  */
-module.exports = function getNewReleaseType() {
-	const conventionalRecommendedBump = require( 'conventional-recommended-bump' );
-	const parserOpts = require( '../changelog/parser-options' );
+module.exports = function getNewReleaseType( transformCommit, isDevPackage ) {
+	const packageJson = getPackageJson();
+	let fromVersion = versions.getLastFromChangelog();
+
+	if ( fromVersion ) {
+		if ( isDevPackage ) {
+			fromVersion = packageJson.name + '@' + fromVersion;
+		} else {
+			fromVersion = 'v' + fromVersion;
+		}
+	}
+
+	const gitRawCommitsOpts = {
+		format: '%B%n-hash-%n%H',
+		from: fromVersion,
+		merges: undefined,
+		firstParent: true
+	};
+
+	const context = {
+		displayLogs: true,
+		packageData: packageJson
+	};
 
 	return new Promise( ( resolve, reject ) => {
-		const options = {
-			whatBump: getNewVersionType
-		};
+		gitRawCommits( gitRawCommitsOpts )
+			.on( 'error', reject )
+			.pipe( conventionalCommitsParser( parserOptions ) )
+			.pipe( concat( ( data ) => {
+				const commits = conventionalCommitsFilter( data );
+				const releaseType = getNewVersionType( commits );
+
+				return resolve( { releaseType } );
+			} ) );
+	} );
+
+	// Returns a type of version for a release based on the commits.
+	//
+	// @param {Array} commits
+	// @returns {String}
+	function getNewVersionType( commits ) {
+		let hasNewFeatures = false;
+		let hasChanges = false;
+
+		for ( const item of commits ) {
+			const singleCommit = transformCommit( item, context );
 
-		conventionalRecommendedBump( options, parserOpts, ( err, response ) => {
```

Comment:
Because `conventionalRecommendedBump` does not allow passing custom options to `git log` command, I had to remove this package and rewrite it. 

Thanks to that I could resolve #96.